docs/tutorials/depth/videos_monodepth2.py

```diff
@@ -0,0 +1,196 @@
+"""02. Predict depth from an image sequence or a video with pre-trained Monodepth2 models
+===========================================================================
+This article will demonstrate how to estimate depth from your image sequence or video stream.
+
+Please follow the `installation guide <../../index.html#installation>`__
+to install MXNet and GluonCV if not yet.
+
+First, import the necessary modules.
+"""
+import os
+import argparse
+import time
+import PIL.Image as pil
+import numpy as np
+
+import mxnet as mx
+from mxnet.gluon.data.vision import transforms
+
+import gluoncv
+from gluoncv.model_zoo.monodepthv2.layers import disp_to_depth
+
+import matplotlib as mpl
+import matplotlib.cm as cm
+import cv2
+
+# using cpu
+ctx = mx.cpu(0)
+
+##############################################################################
+# Prepare the data
+# -----------------
+#
+# In this tutorial, we use one sequence of KITTI RAW datasets as an example.
+# Because the KITTI RAW dataset only provides image sequences, the input format is image sequences in this tutorial.
+#
+# Follow the command to download example data::
+#
+#       cd ~/.mxnet/datasets/kitti/examples
+#       wget https://s3.eu-central-1.amazonaws.com/avg-kitti/raw_data/2011_09_26_drive_0095/2011_09_26_drive_0095_sync.zip
+#       unzip 2011_09_26_drive_0095_sync.zip
+#
+#
+# After getting the dataset, we can easily load images with PIL.
+data_path = "~/.mxnet/datasets/kitti/example/2011_09_26/2011_09_26_drive_0095_sync/image_02/data"
+
+files = os.listdir(data_path)
+files.sort()
+
+raw_img_sequences = []
+for file in files:
+    file = os.path.join(data_path, file)
+    img = pil.open(file).convert('RGB')
+    raw_img_sequences.append(img)
+
+original_width, original_height = raw_img_sequences[0].size
+
+##############################################################################
+# Loading the model
+# -----------------
+#
+# In this tutorial we feed frames from the image sequences into a depth estimation model,
+# then we could get the depth map of the input frame.
+#
+# For the model, we use monodepth2_resnet18_kitti_mono_stereo_640x192 as it is accurate and
```

Comment:
Add quotes around the model name, like `monodepth2_resnet18_kitti_mono_stereo_640x192 `

Reply:
Fixed. Thanks for your reminder!